custom_components/tahoma/light.py

```diff
@@ -56,6 +66,13 @@ def is_on(self) -> bool:
         """Return true if light is on."""
         return self._state
 
+    @property
+    def hs_color(self):
+        """Return the hue and saturation color value [float, float]."""
+        if self.supported_features & SUPPORT_COLOR:
```

Comment:
Maybe better to check if `self._rgb` is set? This function will only be called if `SUPPORT_COLOR` is in `supported_features`.

Reply:
It seems from `kwargs` given to `turn_on` and the `hs_color` property that RGB isn't the standard way. Should we implement hue saturation instead of RGB, and only convert to RGB for the apply_action which is specific to TaHoma?